{{cookiecutter.app_name}}/requirements/dev.txt

```diff
@@ -5,6 +5,19 @@
 pytest==2.8.2
 WebTest==2.0.20
 factory-boy==2.6.0
+ipdb==0.8.1
```

Comment:
Since ipdb and ipython aren't hard requirements for setting up the dev environment and alternatives exist (e.g ptpython, bpython), I think these should be removed from the dev requirements.